src/v2/view-builder/views/IdentifierView.js

```diff
@@ -92,19 +92,21 @@ const Body = BaseForm.extend({
 
   showMessages() {
     /**
-     * Renders a warning callout for unknown user flow
+     * Renders callout for unknown user flow
+     * The callout type is warning unless the message states differently
      * Note: Anytime we get back `messages` object along with identify view
      * we would render it as a warning callout
      * */
     const messagesObj = this.options.appState.get('messages');
     if (messagesObj?.value.length
         && this.options.appState.get('currentFormName') === 'identify') {
-      const content = messagesObj.value[0].message;
+      const displayMessageObj = messagesObj.value[0];
+      const messageType = (displayMessageObj.class || 'warning').toLowerCase();
```

Comment:
i think we should get rid of `warning` callout.. there is no use case to show warning message in this view.. just render the `error` callOut like any other view that overrides it.

Reply:
I don't know enough whether warning should be removed. The change here is just to respect whatever the idx api message type is. So it should work for both warning and error cases.

Reply:
there is no `WARN` type in backend.. it's only `INFO` and `ERROR`.. the comment clarifies that it was something hardcoded.